From d559aed2941b6f40c9bc0bc23b03a389e6fd029a Mon Sep 17 00:00:00 2001
From: Andrew Murray <andrew.murray@arm.com>
Date: Mon, 17 Jun 2019 10:36:01 +0100
Subject: [PATCH 7/8] clocksource/arm_arch_timer: remove unused return type

The function 'arch_timer_set_evtstrm_feature' has no return statement
despite its prototype - let's change the function prototype to return
void. This matches the equivalent arm64 implementation.

fixes: 11e34eca5d0a ("clocksource/arm_arch_timer: Extract elf_hwcap use to arch-helper")
Reported-by: kbuild test robot <lkp@intel.com>
Signed-off-by: Andrew Murray <andrew.murray@arm.com>
(am from https://lore.kernel.org/lkml/20190617093601.34511-1-andrew.murray@arm.com/)
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 arch/arm/include/asm/arch_timer.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/include/asm/arch_timer.h b/arch/arm/include/asm/arch_timer.h
index ae533caec1e9..99175812d903 100644
--- a/arch/arm/include/asm/arch_timer.h
+++ b/arch/arm/include/asm/arch_timer.h
@@ -125,7 +125,7 @@ static inline void arch_timer_set_cntkctl(u32 cntkctl)
 	isb();
 }
 
-static inline bool arch_timer_set_evtstrm_feature(void)
+static inline void arch_timer_set_evtstrm_feature(void)
 {
 	elf_hwcap |= HWCAP_EVTSTRM;
 }
-- 
2.22.0

